FakeTrello/Models/Card.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Web;
+
+namespace FakeTrello.Models
+{
+    public class Card
+    {
+        [Key]
+        public int CardId { get; set; }
```

Comment:
@katerebekah  Would it be something like this?

public TrelloUser Contributer { get; set; }
public TrelloUser Owner { get; set; }